hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/utils/TestResourceLimitCache.java

```diff
@@ -60,25 +58,27 @@ public void testResourceLimitCache()
     resourceCache.remove(4);
 
     GenericTestUtils.waitFor(future::isDone, 100, 1000);
-    // map has the ket 1
+    // map has the key 1
     Assert.assertTrue(future.isDone() && !future.isCompletedExceptionally());
     Assert.assertNotNull(resourceCache.get(1));
 
     // Create a future which blocks to put 4. Currently map has acquired 7
     // permits out of 10
+    ExecutorService pool = Executors.newCachedThreadPool();
     future = CompletableFuture.supplyAsync(() -> {
       try {
         return resourceCache.put(4, "a");
       } catch (InterruptedException e) {
-        e.printStackTrace();
+        return null;
       }
-      return null;
     });
```

Comment:
`pool` is currently unused, I think you intended to pass it to `supplyAsync`:

```suggestion
    }, pool);
```

Reply:
Thanks @adoroszlai for the notification.
After testing, I want to re-apply this modification in a new branch, and I missed it.